card/mod/machines/lib/javascript/decko_navbox.js.coffee

```diff
@@ -1,72 +1,69 @@
 $(window).ready ->
```

Comment:
navbox based on select2 instead of jquery-ui-autocomplete

Reply:
any interface changes?

Reply:
yes, only a little one but I'm not really happy with it. 
select2 doesn't support input fields for ajax request. It has to be a select field. 
That means you don't write in the search field you write in a field below: 
<img width="571" alt="my_deck" src="https://user-images.githubusercontent.com/13584/39922354-0361054c-551f-11e8-8987-2fe07a2386e2.png">